lerna.json

```diff
@@ -1,7 +1,13 @@
 {
-  "lerna": "2.0.0-beta.36",
   "version": "2.3.1",
   "command": {
+    "version": {
```

Comment:
does this automatically run the changelog for us now?

Reply:
Yes, sending you new release process doc. :)

Reply:
cool!